src/main/services/knex.js

```diff
@@ -39,7 +39,132 @@ const init = async () => {
 	}
 };
 
+const sqlUtil = {
```

Comment:
The vast majority of these functions are a bad idea.

There's no compelling reason to wrap the `knex` API in this layer of indirection. It makes debugging harder because we're adding our own custom API on top (is the bug in our helpers, in `knex` or in how we implement the `knex` API?) and it's a very leaky abstraction, since the calling code needs to know about how `knex` will handle the transaction object passed in.

We should get rid of as many of these functions as possible and prefer using the `knex` builder where possible - or implement something like Bookshelf.

Reply:
Will move everything to objection.js